From 6b092b83aca7a47c123415adc1fb83aab86b730e Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Sat, 22 May 2021 10:22:25 +0300
Subject: [PATCH] [SW-40142] boot_desc: fix prefix for RMA desc

Fix prefix in boot_info types related to RMA, to match
the convention as in other fields.

Change-Id: I81aa80be37cba84e2126542367fc817d21221d7b
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/gaudi/rma_info.c b/gaudi/rma_info.c
index ec81fa3..0598267 100644
--- a/gaudi/rma_info.c
+++ b/gaudi/rma_info.c
@@ -370,17 +370,17 @@
 
 	ret = rma_info_get(RMA_HBM_ROW_REPLACEMENT_TABLE, row_rep_table, &size);
 	if (!ret)
-		boot_info_set(HL_BOOT_RMA_ROW_REP_TABLE, row_rep_table, size);
+		boot_info_set(HL_BOOT_INFO_RMA_ROW_REP_TABLE, row_rep_table, size);
 
 	size = MAX_INFO_DESC_DATA;
 	ret = rma_info_get(RMA_HBM_ECC_TABLE, hbm_ecc_table, &size);
 	if (!ret)
-		boot_info_set(HL_BOOT_RMA_HBM_ECC_TABLE, hbm_ecc_table, size);
+		boot_info_set(HL_BOOT_INFO_RMA_HBM_ECC_TABLE, hbm_ecc_table, size);
 
 	size = MAX_INFO_DESC_DATA;
 	ret = rma_info_get(RMA_HEADER_TABLE, hbm_hdr_table, &size);
 	if (!ret)
-		boot_info_set(HL_BOOT_RMA_HDR_TABLE, hbm_hdr_table, size);
+		boot_info_set(HL_BOOT_INFO_RMA_HDR_TABLE, hbm_hdr_table, size);
 }
 
 int rma_table_read(enum rma_tables_type type, struct rma_table **table, size_t *data_len)
